src/wp-includes/class-wp-autoloader.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Class WP_Autoloader
+ *
+ * WordPress Class, Interface and Trait Autoloader
+ *
+ */
+class WP_Autoloader {
+
+	/**
+	 * @var array
+	 */
+	private static $_classmap;
+
+	/**
+	 * WP_Autoloader constructor.
+	 */
+	function __construct() {
+
+		if ( ! defined( 'DISABLE_CORE_AUTOLOADER' ) ) {
```

Comment:
My bad, you are correct. I was looking at other parts of the code.  Will fix next.